src/policies.rs

```diff
@@ -162,113 +176,127 @@ fn commits_to_verify<'a, G: Git>(
     git: &'a G,
     old_commit_id: Oid,
     new_commit_id: Oid,
-) -> Result<Vec<Commit<'a>>, Box<dyn Error>> {
-    if old_commit_id.is_zero() {
-        debug!("New branch detected");
-        git.find_unpushed_commits(new_commit_id)
-    } else {
-        git.find_commits(old_commit_id, new_commit_id)
-    }
+
+    override_tag_filter: &Option<String>,
+) -> Result<Vec<VerificationCommit>, Box<dyn Error>> {
+    git.find_commits(&[old_commit_id], &[new_commit_id], override_tag_filter)
```

Comment:
Double check what happened to the is_zero check.

Reply:
Checked. It just moved into `find_commits`. Relevant block is:
```
for &exclusion in exclusions.iter().filter(|id| !id.is_zero()) {
    revwalk.hide(exclusion)?;
}
```